src/Mod/Part/App/FeatureOffset.h

```diff
@@ -42,6 +42,7 @@ class PartExport Offset : public Part::Feature
     App::PropertyFloat Value;
     App::PropertyEnumeration Mode;
     App::PropertyEnumeration Join;
+    App::PropertyEnumeration Algo;
```

Comment:
Because they share the same GUI class. I don't want to be messing around with dynamic creation of GUI elements. My most recent change has separated AlgoEnum for offset, and offset2D.